1apply-css/about.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>About Sharks</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div>There are over <strong>500</strong> species of sharks.</div>
+    <div>The great white shark can grow up to <strong class="highlight">6.1 meters</strong> long.</div>
+</body>
+
+</html>
```

Comment:
I noticed this "no newline at the end of the file" is being flagged. It's best practice to have a newline at the end of your files and you can actually set VSCode up to do it automatically! [Here's a link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)

Reply:
This is PURE GOLD--thank you so much!!! :)